WooCommerce/WooCommerceTests/ViewRelated/Shipping Label/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsItemViewModelTests.swift

```diff
@@ -0,0 +1,50 @@
+import XCTest
+@testable import WooCommerce
+
+class WooShippingCustomsItemViewModelTests: XCTestCase {
+    private var viewModel: WooShippingCustomsItemViewModel!
+
+    override func setUp() {
+        viewModel = WooShippingCustomsItemViewModel(title: "",
```

Comment:
Missing: `super.setUp()`

Reply:
Thanks, done in https://github.com/woocommerce/woocommerce-ios/pull/14836/commits/21da0ea75faee37e9b4ed80205d384d12aca1632